Userland/Libraries/LibJS/Bytecode/Generator.h

```diff
@@ -202,6 +203,9 @@ class Generator {
     void emit_get_by_id(IdentifierTableIndex);
     void emit_get_by_id_with_this(IdentifierTableIndex, Register);
 
+    // NOTE: To generate bytecode for global variable identifiers, accessing m_next_property_lookup_cache is necessary.
+    friend Bytecode::CodeGenerationErrorOr<void> JS::Identifier::generate_bytecode(Bytecode::Generator& generator) const;
```

Comment:
:yakfused:
```suggestion
    size_t next_property_lookup_cache() { return m_next_property_lookup_cache++; }
```

also, that name is odd, do you mean `m_next_property_lookup_cache_id`?